ArcGISRuntimeSDKQt_CppSamples/Routing/ServiceArea/ServiceArea.qml

```diff
@@ -102,6 +102,19 @@ ServiceAreaSample {
                 else
                     setBarrierMode();
             }
+
+            property int modelWidth: 0
+            width: modelWidth + leftPadding + rightPadding + indicator.width
```

Comment:
in a few places you have the property and width calculation below another onXXXX function like this.

We should have the properties and width settings at the top of the combo-box declaration I think